src/store/reducers/index.ts

```diff
@@ -1,14 +1,16 @@
-import { Reducer } from "redux";
-
 import { roomReducer } from "./Room";
 import { locationReducer } from "./Location";
 import { attendanceReducer } from "./Attendance";
 import { chatReducer } from "./Chat";
 import { sovereignVenueReducer } from "./SovereignVenue";
 import { userProfileReducer } from "./UserProfile";
+import { animateMapReducer } from "./AnimateMap";
+import { VenueTemplate } from "../../types/venues";
 
 // Reducers per VenueTemplate (eg. reducer for playa template)
-export const VenueTemplateReducers: { [key: string]: Reducer } = {};
+export const VenueTemplateReducers = {
+  [VenueTemplate.animatemap]: animateMapReducer,
+};
```

Comment:
Why was the explicit typing of this function removed?

Should be either (matching how it used to be):
```typescript
export const VenueTemplateReducers: { [key: string]: Reducer } = {
  [VenueTemplate.animatemap]: animateMapReducer,
};
```

Or if the typing works, this would seem more explicit:
```typescript
export const VenueTemplateReducers: { [key: VenueTemplate]: Reducer } = {
  [VenueTemplate.animatemap]: animateMapReducer,
};
```

Which could also likely be simplified to just:
```typescript
export const VenueTemplateReducers: Partial<Map<VenueTemplate, Reducer>> = {
  [VenueTemplate.animatemap]: animateMapReducer,
};
```